services/llgen/kilda/llgen/daemon.py

```diff
@@ -0,0 +1,192 @@
+import errno
+import fcntl
+import resource
+import os
+import signal
+import sys
+import time
+import traceback
+
+
```

Comment:
Why is this class needed? What is its goal in life? ... code comments

This is a rather large class, some comments are "extra necessary"